dali/python/nvidia/dali/plugin/mxnet.py

```diff
@@ -229,40 +241,60 @@ def __init__(self,
             "output_names in output_map should be distinct"
         self.output_map = output_map
 
-        super(DALIGenericIterator, self).__init__(
```

Comment:
Because of:
```
class _DALIMXNetIteratorBase(mx.io.DataIter, _DaliBaseIterator):
```
So there is more than one base class so I need to name it here.

Reply:
But `DALIGenericIterator` has one base class: `class DALIGenericIterator(_DALIMXNetIteratorBase):`, so for its init `super()` should be enough, I think. Unless, I don't understand something here. 
Regardless, it's not that big of a deal. It was something that caught my eye.

Reply:
Done